app/data/projects.json

```diff
@@ -751,10 +751,11 @@
     "category": "Operations"
   },
   {
-    "projectName": "minishift",
+    "projectName": "Minishift",
     "projectDescription": "Run Openshift Locally",
-    "projectRepository": "https://github.com/minishift",
-    "category": "Operations"
+    "projectRepository": "https://github.com/minishift/minishift",
+    "projectWebsite": "https://www.openshift.org/minishift/",
```

Comment:
http://minishift.io  should be used instead. as it points to the actual openshift.org/minishift page by redirection

Reply:
Done @gbraad